flows/engine/assets.go

```diff
@@ -64,14 +65,17 @@ func NewSessionAssets(source assets.Source, migrationConfig *migrations.Config)
 		return nil, err
 	}
 
+	fieldAssets := flows.NewFieldAssets(fields)
+	groupAssets, _ := flows.NewGroupAssets(env, fieldAssets, groups)
```

Comment:
Not entirely sure what to do here.. this method can return the list of "broken" groups.. but what should we do with that? My inclination is that we ignore broken groups at this point - we should be a better job of indicating in the RP UI that a dynamic group is broken.